docker-compose.yml

```diff
@@ -1,17 +1,22 @@
-# docker-compose.yml
 version: '3'
 networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: todoapp:2.1.0
     build:
       context: .
       dockerfile: Dockerfile
     ports:
       - "8080:8080"
     environment:
       - PYTHONUNBUFFERED=1
+      - ENGINE=mysql.connector.django
+      - NAME=app_db
+      - USER=app_user
+      - PASSWORD=1234
```

Comment:
Using hardcoded passwords, such as '1234', is a security risk. Consider using environment variables or Docker secrets to manage sensitive information securely.